clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -449,6 +449,7 @@
     "scripts": {
         "vscode:prepublish": "npm run compile",
         "postinstall": "patch-package",
+        "test:integration": "npm run compile && node ./out/test/runTest.js",
```

Comment:
It is mandatory for the integration test that the server.jar is placed correctly. Should I create a script to check that and if not present, compile the server and copy the jar?

Reply:
I think it is important only for a local case. Maybe we should set up an earthly build instead of this script.